frodo.js

```diff
@@ -1,14 +1,14 @@
-const oddNumbers = numbers.filter(num => num % 2 !== 0);
```

Comment:
SCRIPT-This logic might be clearer if you used a switch statement instead of multiple if-else statements.